### ISSUE TITLE
[Client requests] Ask for your missing clients here

### ISSUE THREAD

**Issue by anthonyraymond** [anthonyraymond/joal#99]
## µTorrent
 - [ ] v2.2.1, build 25130

## rTorrent
- [ ] 0.9.8

**Comment by anthonyraymond** [comment]
I'll take a look at it for the refactored version to come in next few months

**Comment by Modged** [comment]
Adding deluge 2.0.3 ? :)

**Comment by MacArony57** [comment]
Hi, can you add the torrent seed-box ID embedded in the Free french ISP modem please ?
I believe latest is Freebox BitTorrent 2.0 depending on the firmware they use. 
others are like this :
…
FX 0.1.0.3        Freebox BitTorrent 1.3
FX 0.1.0.2        Freebox BitTorrent 1.2
FX 0.1.0.1        Freebox BitTorrent 1.1

**Comment by anthonyraymond** [comment]
@Modged Hello, i'm lacking time to add some new deluge atm, but i'll do it soon or later :)

@MacArony57, Sorry but those are way to specific piece of software, hard to analyse, reproduce and i won't be able to keep track of the upgrade since i don't own a freebox.

**Comment by OliverGrace** [comment]
qBittorrent 4.3.9

**Comment by anthonyraymond** [comment]
@OliverGrace Done. Release is coming soon

**Comment by OliverGrace** [comment]
> @OliverGrace Done. Release is coming soon

Appreciate that.

**Comment by JourneyOver** [comment]
Thanks for that @anthonyraymond I have been pretty busy so I haven't really had time any to grab releases for qbt..

**Comment by anthonyraymond** [comment]
No worries 🙂 i feel glass for what you've done already. No need to rush yourself 😉

**Comment by adrenalien** [comment]
deluge 2.0.5 please

**Comment by methbkts** [comment]
Hi @anthonyraymond

Can you please add qBittorrent's 4.4.3 version for consistency with the last version of the client ?

Thanks 🙂

**Comment by JourneyOver** [comment]
@methbkts I have made a PR for the latest version of qbt.

**Comment by Xiseous** [comment]
Please could you add, Transmission 4.0.3 which is currently the latest

**Comment by niawag** [comment]
Hi, for Transmission 4.0.3 client I thought I could try it myself using this: https://github.com/anthonyraymond/joal/blob/master/scripts/bittorrent-client-update-detector/transmission.sh so I made a PR: https://github.com/anthonyraymond/joal/pull/214

**Comment by apsinghnandha** [comment]
Hi could you please update the qbittorrent client to 4.5.4 and also add the list of clients with version being supported. Thanks

**Comment by JourneyOver** [comment]
@apsinghnandha 4.5.4 already exits for qbittorrent client, just go and download it from https://github.com/anthonyraymond/joal/blob/master/resources/clients/qbittorrent-4.5.4.client and drag and drop into your clients folder.

**Comment by apsinghnandha** [comment]
> @apsinghnandha 4.5.4 already exits for qbittorrent client, just go and download it from https://github.com/anthonyraymond/joal/blob/master/resources/clients/qbittorrent-4.5.4.client and drag and drop into your clients folder.

Thanks. All worked out.

**Comment by sharktachi** [comment]
> Hi, for Transmission 4.0.3 client I thought I could try it myself using this: https://github.com/anthonyraymond/joal/blob/master/scripts/bittorrent-client-update-detector/transmission.sh so I made a PR: #214

Is this safe to use?

Is see qBitorrent client gets frequent updates and I would like to ask if it could be the case of Transmission, which client file is at 3.00 while latest official release is at 4.0.4. 

Or if possible a tutorial step by step for non tech-savvy users to generate it with peace of mind it will not mess up at trackers. I am able to capture traffic via Wireshark+npcap and a dummy tracker build but I do not know how to get any further to build it.

Thanks in advance for this great piece of coding.

**Comment by niawag** [comment]
Hi, I'm using it (and the 4.0.4 iteration I made the same way) as is since the 5 of July without problem on two private trackers. I can't be sure it's safe but it doesn't seem too risky.

But I'm also in favor of an how-to for transmission client so the community (me included) can help more.

**Comment by sharktachi** [comment]
> Hi, I'm using it (and the 4.0.4 iteration I made the same way) as is since the 5 of July without problem on two private trackers. I can't be sure it's safe but it doesn't seem too risky.
> 
> But I'm also in favor of an how-to for transmission client so the community (me included) can help more.

Thank you for your feedback.

Let us cross fingers for @anthonyraymond to have a look here and help us with a tutorial once he became active again on the project and launched a great update since may. I am loving the ratio limiter feature so far for not overseeding. It only needs a feature to stop seeding a torrent if you are the only seeder to make it perfect.

**Comment by likeman66** [comment]
Thx to this great project. But is it possible to spoof the download as well?  Once add the .torrent file, it just skill the download process(I mean fake download process) which is quite suspicious and can be easily detected. It would be better for user to choose whether each torrent file uploaded should skip the download process and start seeding directly (the current way) or choose to imitate the download process

**Comment by Xiseous** [comment]
Could someone generated qBit 4.6.5,

And potentially write up a guide on how to generate the pair file from the libtorrent.sh?

Thanks,

**Comment by JourneyOver** [comment]
I'll push a new version up here soon for QBT. Kind of just forgot about things these past couple months as I haven't been torrenting as much since a lot of my shows are on break, and I rarely ever pirate games much anymore.

**Comment by JourneyOver** [comment]
@Xiseous I've pushed up a PR for the latest version (4.6.5) which you can see here -- https://github.com/anthonyraymond/joal/pull/237

5.0.0b1 is a beta release so it is technically not the latest version, the script needs updating on how it does certain things (like making it grab only non-betas/alphas). I've already made those changes on my side, but joal didn't want them so I just never PRd them up.

**Comment by Xiseous** [comment]
> I'll push a new version up here soon for QBT. Kind of just forgot about things these past couple months as I haven't been torrenting as much since a lot of my shows are on break, and I rarely ever pirate games much anymore.

Makes sense, I would be in the same spot if I didn't use something much, I wouldn't be updating it as well

**Comment by Xiseous** [comment]
> @Xiseous I've pushed up a PR for the latest version (4.6.5) which you can see here -- #237
> 
> 5.0.0b1 is a beta release so it is technically not the latest version, the script needs updating on how it does certain things (like making it grab only non-betas/alphas). I've already made those changes on my side, but joal didn't want them so I just never PRd them up.

Thank you!

**Comment by Xiseous** [comment]
@JourneyOver I've been tweaking and fixing up the libtorrent/qbittorrent updater, can you let me know if this file is correct? (I had to upload as .json rather than .client due to restrictions).

[qbittorrent-5.0.5.json](https://github.com/user-attachments/files/22959894/qbittorrent-5.0.5.json)

I've tested this by using the original scripts located, modifying them to look at the new source files for the newer client versions, tested output of script against known good outputs (the clients from the repo itself compared to the auto generated ones as well as newer clients seen in this thread)

The script:
`Usage: ./qbit_analyzer.sh [OPTIONS] [URL]`

`Analyze qBittorrent source code to extract client identification information.`

`Options:`
`    -h, --help              Show this help message`
`    -v, --verbose           Enable verbose output`
`    -f, --force             Force re-download (ignore cache)`
`    -o, --output FORMAT     Output format: text, json, csv (default: text)`
`    --major-version NUM     Get latest stable release for major version (e.g., 4)`
`    --list-releases         List available releases and exit`
`    --force-latest          Use absolute latest release (including pre-releases)`
`    --no-cache              Disable caching`

`Arguments:`
`    URL                     GitHub API tarball URL or release API endpoint`
`                           If not provided, uses latest stable release`

`Examples:`
`    ./qbit_analyzer.sh                                          # Analyze latest stable release`
`    ./qbit_analyzer.sh --major-version 4                        # Latest v4.x stable release`
`    ./qbit_analyzer.sh --list-releases                          # Show available releases`
`    ./qbit_analyzer.sh --output json                            # Output in JSON format`
`    ./qbit_analyzer.sh https://api.github.com/repos/qbittorrent/qBittorrent/tarball/refs/tags/release-4.6.2`

So as per previous comments, this will allow the end user to download pre-release or beta versions and automatically make the .client file for them (which hopefully could be pushed as a PR).

**Comment by JourneyOver** [comment]
@Xiseous yea it looks all correct!

Honestly I was thinking about doing the whole github workflows thing a long time ago so I didn't have to keep doing it manually when I first started doing the updates for qbittorrent before anthony disappeared, but I never did get around to implementing it.

**Comment by Mylloon** [comment]
Transmission 4.0.6

**Comment by trymz** [comment]
Deluge 2.2.0 would be great
